debian/nfd.postinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+if ! ndnsec-get-default >/dev/null
+then # generate keys for this machine
+  ndnsec-keygen /localhost/operator | ndnsec-install-cert -
```

Comment:
Are keys required? We should ship Endless keys, right? Also, you do know that this isn't per-machine -- it will happen during the OSTree build phase, right?